salt/states/sysctl.py

```diff
@@ -79,7 +79,7 @@ def present(name, value, config=None):
                     'The value {0} is set to be changed to {1} '
             return ret
         elif name in configured and name in current:
-            if str(value) == __salt__['sysctl.get'](name):
+            if str(value).split() == __salt__['sysctl.get'](name).split():
```

Comment:
The problem is  the intervening whitespace sysctl inserts in its output.  If you want I can `replace(r'\s+', ' ')`, but I thought that `.split()` was simpler.